### ISSUE TITLE
wmic is no longer present in Windows 11

### ISSUE THREAD

**Issue by tonesto7** [sebhildebrandt/systeminformation#613]
**Describe the bug**

**To Reproduce**
1. used function '...'
2. code snippet '....'
3. start app / code
4. See output/error '....'

**Current Output**
If applicable, add output to help explain your problem.

**Expected behavior**

**Environment (please complete the following information):**
 - systeminformation package version: 5.7.4
 - OS: [e.g. macOS] Window 11
 - Hardware [e.g. MacBook Pro 13] 

**Additional context**
http://www.edtittel.com/blog/windows-11-says-sayonara-wmic.html

I'm seeing the wmic is not recognized on the getNetworkInterfaces()

**Comment by sebhildebrandt** [comment]
@tonesto7 ... OK, was not aware of this ... anyway I replaced already a lot of `wmic` calls, will also replace the few ones that are still in place. Will let you know as soon as I completed this. Thank you!

**Comment by tonesto7** [comment]
Typical MS crap removing things for no reason.
Thanks for the fast response!!!

**Comment by sebhildebrandt** [comment]
@tonesto7 ... well it was marked as `deprecated` already. But I did not know that it now was removed ;-) I need to do all of this in conjunction with #611 ... as I will need to use `PowerShell` commands now. Unfortunately it seems that
- this takes more resources (cpu load) to spin up a powershell shell
- for some reason in my version it does not shut down the session 
... will do this whole thing on Wednesday.
